build.sbt

```
@@ -199,11 +199,10 @@ lazy val node = {
       mappings in Universal ++= directory((resourceDirectory in Compile).value / "chains").map { case (f, name) =>
         f -> s"conf/$name"
       },
-      jdkPackagerJVMArgs := Seq(
-        "-Dconfig.file=." + sep + "conf" + sep + "app.conf",
-        "-Dlogback.configurationFile=." + sep + "conf" + sep + "logback.xml",
-        "-Xss10M"
-      )
+      bashScriptExtraDefines ++= IO.readLines((resourceDirectory in Compile).value / "scripts" / "extra.sh"),
+      bashScriptExtraDefines += """addJava "-Dlogback.configurationFile=${app_home}/../conf/logback.xml"""",
+      batScriptExtraDefines ++= IO.readLines((resourceDirectory in Compile).value / "scripts" / "extra.bat"),
+      batScriptExtraDefines += """call :add_java "-Dlogback.configurationFile=%APP_HOME%\..\conf\logback.xml""""
```

```suggestion
      batScriptExtraDefines += """call :add_java "-Dlogback.configurationFile=%APP_HOME%\\..\\conf\\logback.xml""""
```

I suggest we add escaping because scala will treat \\., \c and \l as special symbols

---

I think that's not the case because it's enclosed in triple quotes